src/lib.rs

```diff
@@ -107,6 +115,10 @@ impl fmt::Display for Error {
             Error::CannotMakeRangeProof => "failed to generate range proof",
             Error::InvalidRangeProof => "failed to verify range proof",
             Error::InvalidGenerator => "malformed generator",
+            Error::InvalidEcdsaAdaptorSignature => "malformed ecdsa adaptor signature",
+            Error::CannotDecryptAdaptorSignature => "failed to decrypt adaptor signature",
+            Error::CannotRecoverAdaptorSecret => "failed to recover adaptor secret",
+            Error::CannotVerifyAdaptorSignature => "failed to verify adadptor signature",
```

Comment:
nit: I can imagine this typo ("adadptor") making somewhen crazy when trying to grep error logs

Reply:
Thanks fixed!